BackEnd/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,15 @@
+spring:
```

Comment:
This works the exact same way as application.yml, except it only applies to files with the `@ActiveProfiles("test")` annotation.

If a file has the active profile set to "ABC", then spring will look for an application-ABC.yml file to pull values from. Same as the comment on application.yml, this is just so you only have to change things in one place.